docs/install.md

```diff
@@ -5,6 +5,18 @@ After the installation process this cluster will be your [management cluster](/g
 with the Zora components installed. 
 So it is recommended to keep it separated from any application workload.
 
+## Setup Requirements
+
+Ensure your system has these programs in order to install and configure Zora:
+
+- Kubernetes >= 1.21.0
```

Comment:
This requirement is about the **management cluster**, where Zora is installed. 

I think would be clearer with this information (that this requirement is only about the management cluster), what do you think?

Because **target clusters** older than 1.21 work.

Reply:
Indeed, will add that.